code/game/gamemodes/changeling/powers/hivemind.dm

```diff
@@ -81,11 +82,18 @@ var/list/datum/dna/hivemind_bank = list()
 
 	var/S = input("Select a DNA absorb from the air: ", "Absorb DNA", null) as null|anything in names
 	if(!S)	return
+
 	var/datum/changelingprofile/chosen_prof = names[S]
 	if(!chosen_prof)
 		return
 
+	for(var/datum/changelingprofile/stored_prof in changeling.stored_profiles)
```

Comment:
Here I'm checking if the selected person isn't already in your stored profiles using their name, as when you channel a DNA it creates a copy which isn't equal to the one in your stored profiles. This meant you could potentially "absorb" the same person and get free changeling points!